pkg/release/changelog/containerlinux.go

```diff
@@ -16,8 +17,14 @@ func parseContainerLinuxChangelog(body []byte, componentVersion string) (string,
 	if err != nil {
 		return "", microerror.Mask(err)
 	}
+
+	b, ok := releases[componentVersion]
+	if !ok {
+		return fmt.Sprintf("Containerlinux release %q was not found in the changelog", componentVersion), nil
```

Comment:
IMO this should return an error in that case. The changelog for containerlinux should never be empty.

Reply:
I don't agree.
If for example we are testing a non stable version of containerlinux the `devctl release` command fails and you have to generate the release templates yourself.
The behaviour is similar when using an unreleased version of an operator for example. The changelog being set is "Not found".
I don't see a reason why it should behave differently with Containerlinux

Reply:
Is something else wrong then? AFAIK every flatcar release - even alpha channel - has release notes:
https://kinvolk.io/flatcar-container-linux/releases-json/releases.json

Reply:
the url was changed (https://github.com/giantswarm/devctl/pull/240)

Reply:
So is this still needed with #240 in place?

Reply:
If you ask me: yes.
But you are clearly against it so I will close it because I have no strong opinions anyway.